test/ptc/tools/cromwell.clj

```diff
@@ -9,9 +10,12 @@
 (defn status
   "Status of the workflow with ID at CROMWELL-URL."
   [cromwell-url id]
-  (->> (str cromwell-url "/api/workflows/v1/" id "/status")
-       (client/get {:headers (misc/get-auth-header!)})
-       :body misc/parse-json-string :status))
+  (let [url (str/join "/" [cromwell-url "api" "workflows" "v1" id "status"])]
+    (->>  {:method       :get           ; :debug true :debug-body true
```

Comment:
A little cruft here?

Reply:
Where?

Reply:
```
; :debug true :debug-body true
```

Reply:
I could delete it, but then someone would have to figure it out again next time we need it.